src/cdk/a11y/focus-trap/focus-trap.ts

```diff
@@ -355,7 +356,8 @@ export class FocusTrap {
 /**
  * Factory that allows easy instantiation of focus traps.
  * @deprecated Use `ConfigurableFocusTrapFactory` instead.
- * @breaking-change for 11.0.0 Remove this class.
+ * @breaking-change 11.0.0
+ * TODO for 11.0.0 Remove this class.
```

Comment:
This isn't the correct way to mark it either since the `TODO` won't be picked up by the breaking changes script.

It should be something like:

```
@breaking-change 11.0.0 To be removed.
```